WordPress/gutenberg
# Add a basic Options modal

Implements half of #10210.

Adds a basic Options modal:

![localhost_8888_wp-admin_post-new php screen recording 2](https://user-images.githubusercontent.com/612155/46160670-4473b600-c27b-11e8-9376-92324fda1916.png)

Not included in this PR is the ability to toggle "advanced panels" AKA metaboxes.

**To test:**

1. Open the editor
2. Open the _More_ drop-down menu in the top right of Gutenberg
3. Select  _Options_
4. Check that each setting works and that they're persisted when the page reloads
